Additional organizer guidance (#12200)
devopsdays/devopsdays-web: content/page/organizing.md

When deciding on appropriate expenditures, check with your local team and reach out to the core team if you'd like guidance. Remember, these funds are intended to serve your community!

> With all of these expenses, your event will likely be handling a lot of money. All major expenditures should be presented to and approved by your organizing team to help prevent conflicts of interest and provide oversight. Many organizing teams have implemented a process where individual organizers pay for expenses directly and request reimbursement from event funds. Reimbursements are only given if one or more additional organizers approve.

Reviewer: 👍

- Industries/Companies represented (share only in aggregate)
- What they get with what level of sponsorship (refer them to your sponsor page or prospectus)
- If they can buy a speaker spot (NO, but they can submit a talk and they can suggest open space topics onsite)

> - If you provide a badge-scanning mechanism (you’re not obligated to, and we do not recommend providing one. For more information see the QR codes section below.)

Reviewer: can this be anchor linked the section?

Author: Added in commit 8b2b8d5

### QR Codes and badge scanning

Sponsors will often ask if attendee badges can be scanned. Using QR codes to receive attendee information is quick and efficient for sponsors, but can be problematic for organizers and raise privacy concerns from attendees. We strongly discourage you from using scannable badges at your event.

There are two primary methods of badge scanning:

The first and most common method used by conferences is to have badges with codes that simply store a UUID. When sponsors scan badges with the designated app, they get the UUID. Later these are given to the organizers and used to generate a list of contacts. Many conferences use this because it protects attendee data and it also forces sponsors to use specific software or devices for an additional cost (more revenue for the event). However, this practice violates one of our DevOpsDays rules: we do NOT ever give out or sell lists with contact details of attendees.

> THe second method is to encode attendee data directly into the QR code (e.g. using the common VCF format). This method does not require organizers to distribute attendee information and allows sponsors to receive the information directly. But it also means that anyone with a QR code scanner can get that information and organizers will need to be extremely cautious that attendee information is not shared without their consent. Should you choose to implement this, the following are required:

Reviewer: ```suggestion
The second method is to encode attendee data directly into the QR code (e.g. using the common VCF format). This method does not require organizers to distribute attendee information and allows sponsors to receive the information directly. But it also means that anyone with a QR code scanner can get that information and organizers will need to be extremely cautious that attendee information is not shared without their consent. Should you choose to implement this, the following are required:
```

- Attendees must be informed that their information will be encoded, including which details (email, phone, etc) will be included.
- Attendees must be informed of and provided a method to opt-out. This could be done during registration and QR codes not printed on the badges of those who opt out, or you could provide stickers to cover QR codes.

> Additionally we recommend that QR codes be printed on the back side of badges and that badges have two points of contact with lanyards to help ensure that QR codes are only visible when an attendee shares it. Sponsors shoujld be reminded that badges may only be scanned with consent from attendees.

Reviewer 2: ```suggestion
Additionally we recommend that QR codes be printed on the back side of badges and that badges have two points of contact with lanyards to help ensure that QR codes are only visible when an attendee shares it. Sponsors must be informed that badges may only be scanned with consent from attendees, and you must enforce this in practice.
```

The second method is to encode attendee data directly into the QR code (e.g. using the common VCF format). This method does not require organizers to distribute attendee information and allows sponsors to receive the information directly. But it also means that anyone with a QR code scanner can get that information and organizers will need to be extremely cautious that attendee information is not shared without their consent. Should you choose to implement this, the following are required:

- Attendees must be informed that their information will be encoded, including which details (email, phone, etc) will be included.
- Attendees must be informed of and provided a method to opt-out. This could be done during registration and QR codes not printed on the badges of those who opt out, or you could provide stickers to cover QR codes.
- Sponsors must be informed that badges may only be scanned with consent from attendees, and you must enforce this in practice.

> Additionally we recommend that QR codes be printed on the back side of badges and that badges have two points of contact with lanyards to help ensure that QR codes are only visible when an attendee shares it.

Reviewer 3: Love this addition!
